docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Concept Glossary
+----------------------------
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
+
+Construct Validity
+^^^^^^^^^^^^^^^^^^
+
+We explore the concept of construct validity, as described in in Measurement and Fairness [#4]_ , and as it relates to sociotechnical contexts. This in intended to illuminate the concept to readers who are working on solutions in sociotechnical contexts and may also be referenced in explorations that detail how to apply the concept to specific scenarios.
+
+Key Terms 
+~~~~~~~~~
+
+- **Sociotechnical context** – A context in which both societal and technical aspects shape who might be harmed by AI systems and how (see fairlearn.ai for more detailed definition).
+
+- **Unobservable theoretical construct** – An idea or concept that is unobservable and cannot be directly measured but must instead be inferred through observable measurements defined in a measurement model. 
```

Comment:
Would it make sense to first make an entry with "construct"? I think some of our users will be unfamiliar with those. At the very least I would definitely add "observable construct", to distinguish between them. I also think it is important to stress that unobservable theoretical constructs are truly unobservable; it's not due to practical constraints. E.g., in theory we might be able to measure alcohol consumption of drivers, but in practice we can't just add a measurement device in every car and force people to use it.

Reply:
Which of the following should we add @hildeweerts: 
- construct
- observable construct
- observable theoretical construct 

And how do you suggest we define it? 

Also, how might we make the suggested emphasis in the example above (could you illustrate by taking the current definition and appending a suggestion)?

Reply:
Hmm, @hildeweerts, I'm not sure I've seen the term "observable construct" used before. To my understanding, almost by definition, constructs are abstractions. What about something like:

"An abstract idea or concept of interest which is not possible to direct measure, but which must instead be inferred through observable measurements.

Reply:
Yup absolutely - I like the language of "observable properties" and "unobservable constructs" from the paper, if that works for others